src/search/pruning/stubborn_sets.cc

```diff
@@ -110,30 +101,8 @@ bool StubbornSets::mark_as_stubborn(int op_no) {
     return false;
 }
 
-void StubbornSets::prune_operators(
+void StubbornSets::prune(
```

Comment:
Looks like a short line to have a line break, but perhaps I'm misjudging.

Reply:
It would be too long otherwise due to the `override`.